main.js

```diff
@@ -0,0 +1,417 @@
+            alt : "Green Tea",
+        },
+    },
+};
+
+let cart_map=new Map();
+let all_ord=[];
+
+// Model
+
+function removeQty(key){
+    cart_map.delete(key);
+}
+
+function subQty(key){
+    if(cart_map.get(key)=="1"){
+        removeQty(key);
+    }
+    else{
+        let temp_num=Number(cart_map.get(key))-1;
+        cart_map.set(key,temp_num);
+    }
+};
+
+function addQty(key){
+    if(cart_map.get(key) === undefined){
+        cart_map.set(key,"1");
+    }
+    else{
+        let temp_num=Number(cart_map.get(key))+1;
+        cart_map.set(key,temp_num);
+    }
+}
+
+function addOredrToPrevious(e){
+    let temp_obj={};
+    temp_obj.date=new Date();
+    temp_obj.id="u01"+all_ord.length.toString();
+    temp_obj.status="Pending";
+    for(let key of cart_map.keys()){
+        temp_obj[key]=cart_map.get(key);
+    }
+    all_ord.unshift(JSON.stringify(temp_obj));
+    cart_map=new Map();
+}
+
+// View
+
+document.querySelector('body').addEventListener("load",DisplayItem());
+
+function DisplayItem () {
+
+    let sec=document.getElementById("LeftContent");
+
+    for(let key in items){
+
+        let sub_sec=document.createElement("section");
+        sub_sec.className="Inline_Flex";
+
+        for(let j in items[key]){
+
+            if(j=="id"){
+
+                let heading=document.createElement("h1");
+                heading.innerHTML=key;
+                heading.id=items[key][j];
+                sec.append(heading);
+
+            }
+            else{
+
+                let temp_div=document.createElement("div");
+                temp_div.className="Create_Box Show_box";
+                
+                let temp_img=document.createElement("img");
+                for(let k in items[key][j]){
+                    temp_img[k]=items[key][j][k];
+                }
+                temp_div.append(temp_img);
+
+                temp_div.append(document.createElement("br"));
+
+                let text_style=document.createElement("span");
+                text_style.className="Item_Name";
+                text_style.innerHTML=j;
+                temp_div.append(text_style);
+                
+                let cart_but=document.createElement("button");
+                cart_but.className="AddButton";
+                cart_but.value=j;
+                cart_but.insertAdjacentHTML('beforeend'," Add to Cart");
+                temp_div.append(cart_but);
+
+                sub_sec.append(temp_div);
+            }
+        }
+
+        sec.append(sub_sec);
+    }
+    
+};
+
+if(cart_map.size)
+    DisplayCart();
+
+function DisplayCart(){
+
+    let table=document.createElement('table');
+    let ctr=1;
+
+    for(let [key,value] of cart_map){
+
+        let row=document.createElement("tr");
+        
+        let data=document.createElement("td");
+        data.className="table_sr_no";
+        data.innerHTML=ctr;
+        ctr++;
+        row.append(data);
+        
+        data=document.createElement("td");
```

Comment:
data=document.createElement("td");
        data.className="table_item_name";;
        data.innerHTML=key;
        row.append(data);

this part is being repeated. Can we move this to a separate helper function?